src/types.rs

```diff
@@ -41,6 +41,26 @@ impl RowEntry {
             expire_ts,
         }
     }
+
+    pub fn estimated_size(&self) -> usize {
+        let mut size = self.key.len();
+        match &self.value {
+            ValueDeletable::Value(v) => size += v.len(),
```

Comment:
Is it possible to do `ValueDeletable::Value(v) | ValueDeletable::Merge(v)` here? Not sure how robust Rust's pattern matching is.

Reply:
it's possible, let me change this :+1:

Reply:
fixed in ce2772a